src/IO/HDF5/HDF5IOHandler.cpp

```diff
@@ -162,6 +162,10 @@ HDF5IOHandlerImpl::createPath(Writable* writable,
         groups.push(node_id);
         for( std::string const& folder : auxiliary::split(path, "/", false) )
         {
+            htri_t found = H5Lexists(groups.top(), folder.c_str(), H5P_DEFAULT);
```

Comment:
const-correctness and comment
```suggestion
            // avoid creation of paths that already exist
            htri_t const found = H5Lexists(groups.top(), folder.c_str(), H5P_DEFAULT);
```